prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,59 @@
+/*       multiply                          */
+
+var number1 = 5;
```

Comment:
make sure to replace all of the `var` declarations with `const` except for the last one as noted in the instructions, that should be declared with `let`